circuit-macros/src/circuit_type/singleprover_circuit_types.rs

```diff
@@ -1,15 +1,15 @@
 //! Groups type and trait definitions built when the `singleprover_circuit`
 //! argument is given to the macro
 
-use proc_macro2::TokenStream as TokenStream2;
+use proc_macro2::{Ident, TokenStream as TokenStream2};
```

Comment:
getting an unused import warning for `Ident` here

Reply:
resolved in a separate pr